pokemonlib.py

```diff
@@ -80,7 +80,7 @@ def __init__(self):
         #logger.debug("Running pidof calcy got code %d: %s", return_code, stdout)
         #self.calcy_pid = stdout.decode('utf-8').strip()
         # cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief", "--pid", self.calcy_pid]
-        cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief"]
+        cmd = ["adb", "-s", await self.get_device(), "logcat", "-T", "1", "-v", "brief" , "MainService:D i:D ClipboardReceiver:D *:S"]
```

Comment:
Does it work without this change? I don't like forcing pokemonlib, which is a lib, to filter exclusively for that string. Idea being making the lib robust and not having to update it when Calcy updates and breaks the script, like we do with ivcheck. :woman_shrugging:

Reply:
It looks like the intent reciver class is moved to a intents sub class, so the correct class parameter -n is -n tesmath.calcy/.intents.IntentReceiver
So I have to revert 0422875 (please reject it) and make a new.
The regex change for RE_CALCY_IV in 2914a03 we need.
0a20514 is optional if you have a noisy phone like me.

Reply:
It did not came out the way I thought it would. 
Still the same PR. But now with and extra commit 66ecea5